### DISCUSSION TITLE
If not RPi4, don't "pacman -Syu" today.

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#561] [Fixed Issues]
- ~~Kernel `linux-raspberrypi* 5.10.73-1` causes failed boot.~~
- ~~DIY rAudio also suffers this.~~

It's fixed.

**Comment by klassikmann** [comment]
@rern 
What a pity! First thing I did this morning - before reading your warning - was a "pacman -Syu".
Restoring the contents of /var/cache/pacman/pkg/linux-raspberrypi4-5.10.64-1-aarch64.pkg.tar.xz was not sufficient.

**Comment by klassikmann** [comment]
I just found at [https://github.com/raspberrypi/linux/issues/4631](url):

> Sorry guys - there was an upstream back-port introduced into 5.10.65 that required more downstream fixups for older platforms than were actually applied (the sdhost driver isn't normally used on Pi 4). Mea culpa.
The remaining patches are in rpi-5.10.y now, and we'll push out an rpi-update build later on.

**Reply by rern** [reply]
It's fixed.

**Comment by klassikmann** [comment]
Confirmed.
